model/aws/sdk-spans.yml

```diff
@@ -297,3 +297,57 @@ groups:
         requirement_level: recommended
       - ref: aws.s3.part_number
         requirement_level: recommended
+  
+  - id: span.aws.sqs.client
+    extends: span.aws.client
+    type: span
+    stability: development
+    span_kind: client
+    brief: "Attributes that exist for SQS request types."
+    attributes:
+      - ref: aws.sqs.queue.url
+        requirement_level: recommended
+      - ref: messaging.destination.name
+        requirement_level: recommended
+
+  - id: span.aws.sns.client
+    extends: span.aws.client
+    type: span
+    stability: development
+    span_kind: client
+    brief: "Attributes that exist for SNS request types."
+    attributes:
+      - ref: aws.sns.topic.arn
+        requirement_level: recommended
+
+  - id: span.aws.kinesis.client
+    extends: span.aws.client
+    type: span
+    stability: development
+    span_kind: client
+    brief: "Attributes that exist for Kinesis request types."
+    attributes:
+      - ref: aws.kinesis.stream_name
+        requirement_level: recommended
+
+  - id: span.aws.stepfunctions.client
+    extends: span.aws.client
+    type: span
```

Comment:
do you need those defined as spans? I don't think any of them is referenced in markdown and given `"Attributes that exist for Step Functions request types.` explanation, it sounds like this group does not describe any span. 

If you want to document them as spans, could you please document them in markdown including span name format and link from [aws-sdk page](https://github.com/open-telemetry/semantic-conventions/blob/78c42c576a25743902192466cf7ff81889bf3630/docs/cloud-providers/aws-sdk.md#aws-service-specific-attributes)?

Reply:
removed them from the span definition.